cli/net.go

```diff
@@ -606,3 +608,86 @@ var NetBlockListCmd = &cli.Command{
 		return nil
 	},
 }
+
+var NetStatCmd = &cli.Command{
+	Name:      "stat",
+	Usage:     "report resource stat for a scope",
+	ArgsUsage: "scope",
```

Comment:
Can we add some `Description` of valid scopes?

Reply:
sure!

Reply:
done.